example/assets/locale/fi-FI.json

```diff
@@ -0,0 +1,9 @@
+{
+  "test": "Näet lisää napauttamalla kuvakkeita",
```

Comment:
Probably nonsense, but the characters should be correctly Finnish. Just copied some copy from the POS project ;)